pkg/supportbundle/supportbundle_test.go

```diff
@@ -42,5 +44,15 @@ func Test_LoadAndConcatSpec(t *testing.T) {
 	if reflect.DeepEqual(fullbundle, bundle3) == false {
 		t.Error("Full bundle and concatenated bundle are not the same.")
 	}
+	// add nil pointer test case
```

Comment:
+1 for adding test coverage

* I'd extract this into a separate test function `Test_LoadAndConcatSpec_WithNil` for example
* Also add a case where both function arguments are `nil` to ensure it test no panic is caused. The current implementation handles that well

Reply:
copy that, and changes in [5f94198](https://github.com/replicatedhq/troubleshoot/pull/1002/commits/5f941983958aa5ec78fc19cbd4552557aeb7a930)